.circleci/config.yml

```diff
@@ -53,6 +65,13 @@ workflows:
               only: master
           requires:
             - plan
+      - validate
+      # - validate:
+      #     filters:
+      #       branches:
+      #         only: master
+      #     requires:
+      #       - apply
```

Comment:
Will change this to only run after `apply` (on `master`) so that and new domains are caught right after merge.